po/ko.po

```diff
@@ -2833,46 +2925,61 @@ msgstr ""
 
 #: src/types-and-values/arithmetic.md:9
 msgid "\"result: {}\""
-msgstr ""
+msgstr "'결과: {}'"
 
 #: src/types-and-values/arithmetic.md:15
 msgid ""
 "This is the first time we've seen a function other than `main`, but the "
 "meaning should be clear: it takes three integers, and returns an integer. "
 "Functions will be covered in more detail later."
 msgstr ""
+"`main` 이외의 함수는 이번이 처음이지만 의미는 명확합니다. 세 개의 정수를 사용"
+"하고 정수를 반환합니다. 함수는 나중에 더 자세히 다루겠습니다."
 
 #: src/types-and-values/arithmetic.md:19
 msgid "Arithmetic is very similar to other languages, with similar precedence."
-msgstr ""
+msgstr "산술은 다른 언어와 매우 유사하며 우선순위가 비슷합니다."
```

Comment:
Thanks, but it would be great if you can suggest further fixes following https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/reviewing-proposed-changes-in-a-pull-request

Reply:
Thanks!

Reply:
Ok, I was not aware of the suggestions.  I'll try to use it but I don't see the button sometimes.